CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ project(gz-sim-yarp-plugins
 
 find_package(YARP REQUIRED COMPONENTS robotinterface os)
 find_package(YCM REQUIRED)
+find_package(ICUB QUIET)
```

Comment:
Why we added this?

Reply:
My bad, I said to @martinaxgloria to add find_package but actually since it is a run-time and not compile-time dependency this is not needed